templates/leaderboard/includes/list.html

```diff
@@ -4,16 +4,39 @@
     <div class="break">
         <hr>
     </div>
-    <h3>{{ name }}</h3>
-    <ul>
-        {% for investor, profile_context in investors %}
-            <li>
-                #{{ forloop.counter }}:
-                <a href="{{ investor.public_profile_url }}">
-                    <img class="profile-crop" src="{{ investor.avatar_url }}" alt="{{ investor.get_display_name }}" />
-                    {{ investor.get_display_name }} (${{ profile_context.total_earned|floatformat:2|intcomma }})
-                </a>
-            </li>
-        {% endfor %}
-    </ul>
+    <div class="center">
+        <div class="icons">
+            <h3>{{ name }}</h3>
+            <ul>
+                {% for investor, profile_context in investors %}
+                    <li>
+                    <a href="{{ investor.public_profile_url }}">
+                    <div class="leaderboard-padding button">
+                        <img class="profile-crop" src="{{ investor.avatar_url }}" alt="{{ investor.get_display_name }}" />
+                        <br/>
+                        <h5 class="investor-name">#{{ forloop.counter }}: {{ investor.get_display_name }}</h5>${{ profile_context.total_earned|floatformat:2|intcomma }}
+                    </div>
+                    </a>
+                    </li>
+                {% endfor %}
+            </ul>
+        </div>
+
+        <div class="icons">
```

Comment:
Oh yeah, that's right. Okay, cool. I'll apply a commit to this branch with that change.